shared/src/commonMain/kotlin/com/github/jetbrains/rssreader/app/FeedStore.kt

```diff
@@ -10,6 +10,8 @@ import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.launch
+import kotlin.js.ExperimentalJsExport
+import kotlin.js.JsExport
```

Comment:
obsolete?

Reply:
totally, I have to say I am a little bit lost with the `declaration is experimental and its usage should be marked with '@kotlin.js.ExperimentalJsExport` message. IntelliJ warns about any usage of the exported declaration. Can I safely ignore it ?